File: pa-sellergoods-interface/src/main/java/com/xiao/sellergoods/service/BrandService.java
package com.xiao.sellergoods.service;

import com.xiao.pojo.TbBrand;
import com.xiao.servcie.BaseService;

import java.util.List;
import java.util.Map;

/**
 * Created by travelround on 2018/12/6.
 */
public interface BrandService extends BaseService<TbBrand,Long> {
    List<Map> selectOptionList();
    /**
     * 返回全部列表
     * @return
     */
/*    List<TbBrand> findAll();
    PageInfo<TbBrand> findPage(int pageNum, int pageSize);
    int add(TbBrand tbBrand);
    TbBrand findOne(long id);
    int update(TbBrand tbBrand);
    int delete(long[] ids);
    PageInfo<TbBrand> search(Integer page, Integer rows, TbBrand tbBrand);*/
}

File: pa-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.findAll = function () {
        return $http.get('../brand/findAll.do');
    };

    this.findPage = function (page, rows) {
        return $http.get('../brand/findPage.do?page=' + page + '&rows=' + rows)
    };

    this.findOne = function (id) {
        return $http.get("../brand/findOne.do?id="+id)
    };

    this.dele = function (list) {
        return $http.get("../brand/delete.do?ids="+list)
    };

    this.search=function(page,rows,entity){
        return $http.post('../brand/search.do?page='+page+"&rows="+rows, entity)
    };

    this.add = function (entity) {
        return $http.post("../brand/add.do",entity)
    };

    this.update = function (entity) {
        return $http.post("../brand/update.do",entity)
    };
});